package.json

```diff
@@ -74,6 +74,7 @@
     "ember-cli-normalize-entity-name": "^1.0.0",
     "ember-cli-preprocess-registry": "^3.1.2",
     "ember-cli-string-utils": "^1.1.0",
+    "ember-octane-blueprint": "ember-cli/ember-octane-blueprint#8/head",
```

Comment:
I don't think it's a good idea to have git dependencies here

Reply:
I think the plan is to use a version once the related PR is merged and published. I do not have permission to do that.

Reply:
still seems strange to me to have it as a dependency tbh. can't we download/clone the blueprint on demand instead?

Reply:
@Turbo87 Not sure. I open the discussion at https://github.com/ember-cli/ember-octane-blueprint/issues/7, and @NullVoxPopuli agreed on this strategy.

As far as I talked with @rwjblue, the requirements were:

1) we wanted to have an independent repo for the `octane` blueprints that it includes both the  `octane-app` and `octane-addon` blueprints where we could iterate quickly these blueprints.

2) it should facilitate an easy path for users to try octane. With this approach, any user that install the latest `ember-cli` could run:

```
ember new my-app -b octane-app
ember addon my-addon -b octane-addon
```

3) we want to create tests for the octane blueprints, but we wanted to reuse when possible the existing MU blueprint tests to avoid the duplication of both test suites. I commented a possible approach at https://github.com/ember-cli/ember-octane-blueprint/issues/7#issuecomment-450040740

> can't we download/clone the blueprint on demand instead?

What is your suggestion?

Reply:
@ppcano I may have misunderstood before, but idk if we want to include the octane blueprint by default quite yet. most people won't be using it, so it'd just be bloat on their system.

I think e-cli should clone the blueprint repo if it doesn't already exist on their system (this'll also help get away from only using published npm packages)